demos/gplates_global/gplates_global.py

```diff
@@ -141,7 +141,7 @@
 # directory. Below, we verify the required paths in this directory and
 # ensure they exist:
 
-muller_2022_files = obtain_Muller_2022_SE(".")
+muller_2022_files = obtain_Muller_2022_SE_v1_2(".")
```

Comment:
Given a few lines above, we tell the user to:
# Download and unzip this file into the `./gplates_files`
Should "." here be replaced by "./gplates_files" for consistency?